python/PiFinder/main.py

```diff
@@ -100,10 +126,14 @@ def get_sleep_timeout(cfg):
     return sleep_timeout
 
 
-def main(script_name=None):
+def main(script_name, fakehardware, fakecamera, notmp):
     """
     Get this show on the road!
     """
+    init_display(fakehardware)
+    setup_dirs()
+    if not fakehardware:
+        init_keypad()
```

Comment:
I'd like to contain the fake switches in the main class, where we can instantiate all the fake classes so that the rest of the app should not be aware about this. Not sure if feasible, we'll see after I do keyboard and camera